clients/admin-ui/__tests__/index.test.tsx

```diff
@@ -1,22 +0,0 @@
-import Home from "../src/pages";
-import { mockNextUseRouter, render, screen } from "./test-utils";
-
-// skipping while configWizardFlag exists
-describe("Home", () => {
```

Comment:
🧵 Each of the tests I deleted were testing a mocked version of the authentication rules that are now covered by the cypress test. The cypress flow tests how the interaction between localStorage, the redux store, and the actual Next routing will function. In my experience, that's going to a give us a lot more confidence than maintaining these mocks.

(We may get rid of `@testing-library/react` based test altogether, seeing as we have Cypress [component testing](https://github.com/ethyca/fides/pull/1106).)